heartcheck.gemspec

```diff
@@ -17,17 +17,29 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'rack', '>= 1.4.0', '< 2.1'
+  if RUBY_VERSION >= '2.6.0'
+    spec.add_runtime_dependency 'rack', '~> 2.2'
+  else
+    spec.add_runtime_dependency 'rack', '>= 1.4.0', '< 2.1'
```

Comment:
By doing this, any version between 2.1 and 2.19999  will not be accepted. Is this correct?

Maybe this 2nd condition should be `< 2.2`, isn't it?

Reply:
I had based on the versions in the gemfile-old 
but I will test this gap, suddenly I don't even need this if